pkg/orchestrator/scanwatcher/watcher.go

```diff
@@ -399,24 +403,40 @@ func (w *Watcher) reconcileInProgress(ctx context.Context, scan *models.Scan) er
 
 	if *scan.Summary.JobsLeftToRun <= 0 {
 		if failedAssetScans > 0 {
-			scan.State = utils.PointerTo(models.ScanStateFailed)
-			scan.StateReason = utils.PointerTo(models.ScanStateReasonOneOrMoreAssetFailedToScan)
+			scan.Status = models.NewScanStatus(
+				models.ScanStatusStateFailed,
+				models.ScanStatusReasonAssetScanFailed,
+				utils.PointerTo(
+					fmt.Sprintf(
+						"%d succeeded, %d failed out of %d total asset scans",
+						*assetScans.Count-failedAssetScans,
+						failedAssetScans,
+						*assetScans.Count,
+					),
+				),
```

Comment:
We could avoid repetition of this code block if we define this message where it was previously defined